opm/models/discretization/common/tpfalinearizer.hh

```diff
@@ -569,7 +570,7 @@ private:
         if (anyFlores) {
             floresInfo_.reserve(numCells, 6 * numCells);
         }
-        if (enableDispersion) {
+        if (enableDispersion || enableMICP) {
```

Comment:
if constexpr

Reply:
Many thanks @akva2 for your comments, the latest commit attempts to address them. Regarding this comment, only `enableMICP` can be checked with constexpr, since `enableDispersion` depends on if the deck has the `DISPERC` keyword. Please let me know if the way this is handle in the latest commit is ok, or if this needs to be modified.

Reply:
yeah, in that case revert to prior solution. you cannot combine if and if constexpr in one statement.
since we need to do a runtime evaluation anyways for the one case, manually duplicating code so we can if constexpr it out for the other makes no sense.

Reply:
Thanks, now this is set back, I will start a jenkins then.